src/router.js

```diff
@@ -1,8 +1,9 @@
-import { createRouter, createWebHashHistory } from 'vue-router';
+import { createRouter, createWebHistory } from 'vue-router';
+
+import GameView from '@/views/GameView.vue';
 import HomeView from '@/views/HomeView.vue';
-import LoginView from '@/views/LoginView.vue';
 import LobbyView from '@/views/LobbyView.vue';
-import GameView from '@/views/GameView.vue';
+import LoginView from '@/views/LoginView.vue';
```

Comment:
There are merge conflicts here because we just wrapped up a significant restructuring of the organization of frontend files. Sorry this got caught in the crossfire. Essentially we've combined views and components into a new src/routes folder with one subfolder per page that includes both the Page/View and a subfolder with all the components for the page. So all the files that were previously in `src/views` are now in `src/routes` in one subfolder per page e.g. `src/routes/game/GameView.vue` and `src/routes/lobby/LobbyView.vue` etc